platform/src/os/web/web.js

```diff
@@ -451,21 +451,27 @@ export class WasmWebBrowser extends WasmBridge {
     }
     
     alloc_thread_stack(context_ptr, timer) {
-        let tls_size = this.exports.__tls_size.value;
```

Comment:
the problem is that wasm-bindgen removes [`__tls_size`](https://github.com/rustwasm/wasm-bindgen/blob/4765448545d7bfc0223fd52f25d966b0bfbf8004/crates/threads-xform/src/lib.rs#L140)